packages/wrangler/e2e/assets-multiworker.test.ts

```diff
@@ -349,7 +349,9 @@ describe.each(
 				});
 			});
 
-			failsIf(style === "dev registry")(".fetch() existing asset", async () => {
+			failsIf(
+				style === "dev registry" && !wranglerDev.includes("--x-assets-rpc")
```

Comment:
this is good! this tells us that the Proxy Worker solved the issue for these use cases